src/NuGet.Clients/NuGet.CommandLine/Commands/DeleteCommand.cs

```diff
@@ -17,6 +17,9 @@ public class DeleteCommand : Command
         [Option(typeof(NuGetCommand), "CommandApiKey")]
         public string ApiKey { get; set; }
 
+        [Option(typeof(NuGetCommand), "DeleteCommandNoServiceEndpointDescription")]
```

Comment:
Is it possible to use the same string resource for both `DeleteCommand` and `PushCommand`? The string should be the same.

Reply:
As @joelverhagen suggested, use the same resource.

Reply:
Sure, it makes sense.